build_projects/shared-build-targets-utils/Utils/BuildVersion.cs

```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
-        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc3-{CommitCountString}";
+        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-{ReleaseSuffix}-{CommitCountString}";
         public string ProductionVersion => $"{Major}.{Minor}.{Patch}";
```

Comment:
This is not intentionally rc3, I think you need to change this (just found it)

https://github.com/dotnet/core-setup/blob/master/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj#L50